src/main/webapp/data-visualization.js

```diff
@@ -228,38 +231,41 @@ function getMinAndMaxPopulation(populationArray) {
   return {max: max, min: min};
 }
 
+
 // Takes in mapsData object which has a data structure that maps
 // counties to populations, a max population, and a min population.
 // Initializes the geoJson and adds multiple event listeners.
-async function displayCountyGeoJson(mapsData, stateNumber) {
+async function displayCountyGeoJson(mapsData, description, stateNumber, geoData, color) {
   const map = new google.maps.Map(document.getElementById('map'), {
     zoom: stateInfo[stateNumber].zoomLevel,
     center: {lat: stateInfo[stateNumber].lat, lng: stateInfo[stateNumber].lng},
   });
 
-  const countyToPopMap = mapsData.map;
+  countyToPopMap = mapsData.map;
   const maxPopulation = mapsData.maxValue;
   const minPopulation = mapsData.minValue;
-  const colorScale = chroma.scale(['white', 'blue']).domain([minPopulation,
+  const minColor = chroma(color).brighten(2);
+  const maxColor = chroma(color).darken(2);
+  const colorScale = chroma.scale([minColor, maxColor]).domain([minPopulation,
     maxPopulation]);
 
-  const geoData = await getGeoData(stateNumber, true);
   map.data.addGeoJson(geoData);
   map.data.forEach(function(feature) {
     map.data.setStyle((feature) => {
       return {
         fillColor: colorScale(countyToPopMap[feature.j.name]).toString(),
+        fillOpacity: 0.4,
```

Comment:
I'd be interested in also seeing what 0.5 and 0.6 looked like - have you tried them out?

Reply:
Changed to .5. At .6 it becomes a little difficult to see text for cities inside the counties.